script/release/update-version-all-community-settings.xml

```diff
@@ -24,6 +24,11 @@
           <name>KIE Staging Repository Group</name>
           <url>https://repository.jboss.org/nexus/content/groups/kie-group/</url>
         </repository>
+        <repository>
+          <id>kie-repository-group</id>
+          <name>KIE Repositories</name>
+          <url>https://repository.jboss.org/nexus/content/repositories/kie</url>
```

Comment:
Nit-picking, but this is technically not a group, but a specific single repository. So perhaps `kie-internal-repository` and `KIE internate repository` would be better. However, feel free to ignore this, it is just a minor cosmetic fix.